TinkerWorX.SharpCraft.Launcher/Common.cs

```diff
@@ -13,17 +13,22 @@ namespace TinkerWorX.SharpCraft.Launcher
 {
     internal static class Common
     {
-        public static bool IsInitialized { get; private set; }
```

Comment:
The changes you made here should be a PR on their own. When doing version control, you should try to compartmentalize each commit/PR such that they all have meaningful messages. Besides that, these changes were recently made, and were the result of a bug in Visual Studio. The accessors should be on the same line. Also, the use of explicit names will be changed, so aliases are used exclusively, since this is the most common standard.